CHANGELOG.md

```diff
@@ -4,6 +4,58 @@ Releases
 This page documents the releases of Sleeper. Performance figures for each release
 are available [here](docs/13-system-tests.md#performance-benchmarks)
 
+## Version 0.20.0
+
+This contains the following improvements:
```

Comment:
Can we note that this is a breaking change, in the sense that it is not possible to upgrade a v0.19.0 deployment to this version?